core/common/assets/js/api/core/data.js

```diff
@@ -474,44 +480,110 @@ export default class Data extends Commands {
 		this.commandFormats[ command ] = format;
 	}
 
+	/**
+	 * Function create().
+	 *
+	 * Run a command, that will be translated as endpoint for creating new data.
+	 *
+	 * @param {string} command
+	 * @param {*} data
+	 * @param {{}} query
+	 * @param {{}} options
+	 *
+	 * @returns {*} result
+	 */
 	create( command, data, query = {}, options = {} ) {
 		return this.run( 'create', command, { query, options, data } );
 	}
 
+	/**
+	 * Function delete().
+	 *
+	 * Run a command, that will be translated as endpoint for deleting data.
+	 *
+	 * @param {string} command
+	 * @param {{}} query
+	 * @param {{}} options
+	 *
+	 * @returns {*} result
+	 */
 	delete( command, query = {}, options = {} ) {
 		return this.run( 'delete', command, { query, options } );
 	}
 
+	/**
+	 * Function get().
+	 *
+	 * Run a command, that will be translated as endpoint for getting data.
+	 *
+	 * @param {string} command
+	 * @param {{}} query
+	 * @param {{}} options
+	 *
+	 * @returns {*} result
+	 */
 	get( command, query = {}, options = {} ) {
 		return this.run( 'get', command, { query, options } );
 	}
 
+	/**
+	 * Function update().
+	 *
+	 * Run a command, that will be translated as endpoint for updating data.
+	 *
+	 * @param {string} command
+	 * @param {*} data
+	 * @param {{}} query
+	 * @param {{}} options
+	 *
+	 * @returns {*} result
+	 */
 	update( command, data, query = {}, options = {} ) {
 		return this.run( 'update', command, { query, options, data } );
 	}
 
+	/**
+	 * Function options().
+	 *
+	 * Run a command, that will be translated as endpoint for requesting options/information about specific endpoint.
+	 *
+	 * @param {string} command
+	 * @param {{}} query
+	 * @param {{}} options
+	 *
+	 * @returns {*} result
+	 */
 	options( command, query, options = {} ) {
 		return this.run( 'options', command, { query, options } );
 	}
 
-	/**
-	 * @param {ComponentBase} component
-	 * @param {string} command
-	 * @param callback
-	 */
-	register( component, command, callback ) {
-		super.register( component, command, callback );
+	register( component, command, context ) {
+		// Support inline-data commands.
+		if ( 'string' === typeof context ) {
+			const CommandDataInline = CommandData,
+				endpointFormat = context;
+
+			CommandDataInline.getEndpointFormat = () => endpointFormat;
+
+			// Since '$e.commands.run
+			context = ( args ) => ( new CommandDataInline( args ).run() );
```

Comment:
You are very right about `importCommands` but we cannot copy constructor, and use `Object.assign()`, since:
```
Class = class Test{};
ClassRef = Object.assign( {}, Class );
classInstance = new ClassRef(); // ClassRef is not a constructor.
```

Reply:
You are right about `importCommands` we cannot use copy constructor since:
```
Class = class Test{};
ClassRef = Object.assign( {}, Class );
classInstance = new ClassRef();

```